video/decode/vd_lavc.c

```diff
@@ -143,6 +144,9 @@ static const struct vd_lavc_hwdec *const hwdec_list[] = {
     &mp_vd_lavc_vaapi,
     &mp_vd_lavc_vaapi_copy,
 #endif
+#if HAVE_D3D11VA_HWACCEL
+    &mp_vd_lavc_d3d11va_copy,
+#endif
```

Comment:
Is it really ok to place it here? This will prefer d3d11 over d3d9, so copying is preferred over directed GL or ANGLE interop.